src/components/boardPage/CreateTaskModal/CreateTaskModal.tsx

```diff
@@ -40,22 +40,26 @@ export const CreateTaskModal = () => {
   }
 
   function handleCreateSubmition(e: React.FormEvent) {
-    e.preventDefault();
-    createTask(
-      +formData.idTaskList,
-      formData.description,
-      +formData.points,
-      formData.title
-    );
-    resetFormData();
-    changeCreateTaskModalVisibility(false);
+    try {
+      e.preventDefault();
```

Comment:
instead of ´preventDefault, return ´false´ at the end of the function.

Reply:
Change done 👍